internal/bminventory/inventory.go

```diff
@@ -1991,14 +1991,16 @@ func calculateHostNetworks(log logrus.FieldLogger, cluster *common.Cluster) []*m
 			continue
 		}
 		for _, intf := range inventory.Interfaces {
-			for _, ipv4Address := range intf.IPV4Addresses {
-				_, ipnet, err := net.ParseCIDR(ipv4Address)
-				if err != nil {
-					log.WithError(err).Warnf("Could not parse CIDR %s", ipv4Address)
-					continue
+			for _, addresses := range [][]string{intf.IPV4Addresses, intf.IPV6Addresses} {
```

Comment:
Won't it be more intuitive to join the two slices and avoid the nested loop? `for _address := range join(intf.IPV4Addresses, intf.IPV6Addresses...) {`

Reply:
join or append?

Reply:
`append` of course, sorry. Updated the comment. I tested it and it works.